limited_area/region_spec.py

```diff
@@ -22,6 +22,14 @@ def create_bound_method(func, obj):
         return types.MethodType(func, obj, obj.__class__)
 
 
+def normal(lon):
+    if lon > np.pi:
+        return lon - 2.0 * np.pi
+    elif lon < -np.pi:
+        return lon + 2.0 * np.pi
+    else:
+        return lon
+
```

Comment:
I think we can delete the `normal` function -- I had been using this for debugging, but forgot to remove it.